site/en/guide/saved_model.ipynb

```diff
@@ -321,7 +321,7 @@
         "nohup tensorflow_model_server \\\n",
         "  --rest_api_port=8501 \\\n",
         "  --model_name=mobilenet \\\n",
-        "  --model_base_path=\"/tmp/mobilenet\" >server.log 2>\u00261\n",
+        "  --model_base_path=\"/tmp/mobilenet\" >server.log 2>&1\n",
```

Comment:
I bet internal Colab will put these back if we don't add them to the copybara import script.

Reply:
it will, and I agree we should (or at least it would be nice)

Reply:
cl/314171711